src/ResourceManager/DataLakeStore/Commands.DataLakeStore/Commands/EnableAzureRmDataLakeStoreKeyVault.cs

```diff
@@ -0,0 +1,40 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.DataLakeStore.Models;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.DataLakeStore
+{
+    [Cmdlet(VerbsLifecycle.Enable, "AzureRmDataLakeStoreKeyVault")]
+    [Alias("Enable-AdlStoreKeyVault")]
+    public class EnableAzureDataLakeStoreKeyVault : DataLakeStoreCmdletBase
+    {
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 0, Mandatory = true,
+            HelpMessage = "The Data Lake Store account to enable the Key Vault for")]
+        [ValidateNotNullOrEmpty]
```

Comment:
This is a break from the pattern that we have been using for other ADLA and ADLS commands. Is it expected that we should include a "Name" alias as well now? I am ok with adding it here right now, just want to make sure that I know it should be added to the other cmdlets in the future as well.